File: monitors/disk_space/src/main/resources/format.py
import sys
import json
import re

# "Filesystem" "Size" "Used" "Avail" "Use%" "Mounted on"

f = open(sys.argv[1], "r")
x = f.read()
f.close

j = json.loads(x)


def split_str(y):
    row = re.split(r"\s+", y)
    return {"Filesystem": row[0], "Size": int(row[1]), "Used": int(row[2]), "Avail": int(row[3]), "Ued%": int(row[4][:-1]), "Mounted on": row[5]}


def format_data():
    return list(map(split_str, j))

File: monitors/tablespaceSpace/src/main/scala/Format.scala
object Format {
  def formatLine(result: String): FileSystem = {
    val row = result.split("\\s+")
    FileSystem(row(0), row(1).toInt, row(2), row(3), row(4).dropRight(1), row(5))
  }

}

case class FileSystem(Filesystem: String, Size: Int, Used: String, Avail: String, `Ued%`: String, `Mounted on`: String)

case class Server(name: String, cron: String)

case class ConfigAlarm()

case class ProcessCollect(result: String, env: Map[String, String])